freestar-plugin-react-native.podspec

```diff
@@ -15,6 +15,6 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://gitlab.com/freestarcapital/freestar-plugin-react-native.git", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m}"
 
-  s.dependency 'FreestarAds', '~> 4.0'
+  s.dependency 'FreestarAds-PreRelease', '~> 5.15.0-beta-4'
```

Comment:
@carlosalcala-freestar we don't have a pre-release plugin, so unfortunately, I can't approve this podspec change.

Reply:
@maddapper I have restored to the latest Release version, once we have the Core with the Thumbnail Release ready will update this as well.
FYI @kkawai